apps/src/sites/studio/pages/teacher_dashboard/parent_letter.js

```diff
@@ -28,6 +29,7 @@ const store = getStore();
 store.dispatch(setCurrentUserName(scriptData.userName));
 store.dispatch(setSections(scriptData.sections));
 store.dispatch(setSection(scriptData.section));
+store.dispatch(selectSection(scriptData.section.id));
```

Comment:
This is an impressively good example of why the "ducks/dux module" redux pattern is considered an antipattern by some.  Good candidate for near-future refactoring.